Description: Upstream changes introduced in version 1.8.2-0ubuntu0
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 tahoe-lafs (1.8.2-0ubuntu0) natty; urgency=low
 .
   * Update to version 1.8.2
     - Improved performance and fault tolerance of downloads
     - Increased logging
     - Fixed Twisted 10.2 compatitibility
   * Update standards version to 3.9.1
 .
 The person named in the Author field signed this changelog entry.
Author: Paul Hummer <paul@ubuntu.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- tahoe-lafs-1.8.2.orig/src/allmydata_tahoe.egg-info/requires.txt
+++ tahoe-lafs-1.8.2/src/allmydata_tahoe.egg-info/requires.txt
@@ -8,4 +8,4 @@ Nevow >= 0.6.0
 pycrypto == 2.0.1, == 2.1.0, >= 2.3
 pyasn1 >= 0.0.8a
 mock
-pycryptopp >= 0.5.20
\ No newline at end of file
+pycryptopp >= 0.5.14
\ No newline at end of file
--- tahoe-lafs-1.8.2.orig/src/allmydata_tahoe.egg-info/SOURCES.txt
+++ tahoe-lafs-1.8.2/src/allmydata_tahoe.egg-info/SOURCES.txt
@@ -1,4 +1,3 @@
-
 .darcs-boringfile
 COPYING.GPL
 COPYING.TGPPL.html
